File: python/Number of Steps to Reduce a Number to Zero.py
class Solution(object):
    def numberOfSteps (self, num):
        """
        :type num: int
        :rtype: int
        """
        count = 0

        while(num):
            if num%2 != 0:
                count += 1
                num = num - 1
            if num>0:
                num = num//2
                count += 1

        return(count)

File: c/Number of Steps to Reduce a Number to Zero.c
int numberOfSteps (int num){
    int count = 0;
	while(num){
		if(num%2 != 0){
			count++;
			num = num - 1;
		}
		if (num>0){
			num = num/2;
			count++;
		}
	}
	return count;
}
